scanners/nessus/nessus_none.py

```diff
@@ -93,6 +102,25 @@ def scan_id(self) -> int:
             raise RuntimeError("scan_id is None")
         return self._scan_id
 
+    @generic_authentication_factory()
+    def authentication_factory(self):
+        """This is the default function, attached to error reporting"""
+        raise RuntimeError(f"No valid authenticator found for {self}")
+
+    @authentication_factory.register(None)
+    def authentication_set_anonymous(self):
+        """No authentication: don't do anything"""
+        logging.info("NOT configured with any authentication")
```

Comment:
Maybe some more context for this message? Maybe "Nessus scan not configured with any auth"